gh_actions_exporter/main.py

```diff
@@ -37,11 +36,11 @@ async def list_status():
 
 @app.delete("/status/removes")
 async def remove_old_status():
-    clean_status()
+    await clean_status()
```

Comment:
```
@repeat_every(seconds=10)
def clean_status() -> None:
    metrics.cleaner()
```

I am not sure this works

Reply:
In my case i had to add `@app.on_event("startup")` or it does repeat at all

Reply:
you mean, does not repeat at all right? gonna add it

Reply:
added